t/test_pl/like_unlike.t

```diff
@@ -0,0 +1,23 @@
+#!./perl
+
+BEGIN {
+    chdir 't' if -d 't';
+    require './test.pl';
```

Comment:
Should add
```perl
    set_up_inc('../lib');
```
here to make `./perl t/test_pl/like_unlike.t` find `Config` in `fresh_perl()`.